addons/account/migrations/12.0.1.1/openupgrade_analysis.txt

```diff
@@ -14,6 +42,18 @@ account      / account.chart.template   / cash_account_code_prefix (char): now r
 account      / account.chart.template   / company_id (many2one)         : DEL relation: res.company
 account      / account.chart.template   / transfer_account_code_prefix (char): NEW required
 account      / account.chart.template   / transfer_account_id (many2one): DEL relation: account.account.template, required
+account      / account.financial.report / account_ids (many2many)       : DEL relation: account.account
```

Comment:
Why is it here all this extra stuff?

Reply:
It was removed in https://github.com/odoo/odoo/commit/e51b4f659fc5e4253d3156754ada21166c8d1bf8